### ISSUE TITLE
aetr regression

### ISSUE THREAD

**Issue by radare** [radare/radare2#2599]
Happened in https://github.com/radare/radare2/commit/24a27ecfcb180156c68840ab2a4c6563fe29ee79 for t.esil/cmd_aetr
```
[BR]  8  cmd_aetr: flags

Command:  /usr/bin/radare2 -e scr.color=0 -N -q -i /tmp/r2-regressions//cmd_aetr-rad.MttIhb   malloc://1024 > /tmp/r2-regressions//cmd_aetr-out.8xLJFD 2> /tmp/r2-regressions//cmd_aetr-err.co0WZt
File: malloc://1024
Script: aetr 0,0x2147a8,rip,+,[8],==,%z,zf,=,%b64,cf,=,%p,pf,=,%s,sf,=
Diff: --- /tmp/r2-regressions//cmd_aetr-exp.8hiIup	2015-05-21 02:07:55.000000000 +0200
+++ /tmp/r2-regressions//cmd_aetr-out.8xLJFD	2015-05-21 02:07:55.000000000 +0200
@@ -29,8 +29,8 @@
 0005.0e:      XOR      V_26:32 ,      V_20:32 ,      V_27:32
 0005.0f:      AND       0x1:32 ,      V_27:32 ,      V_28:32
 0005.10:      NOT      V_28:32 ,              ,      V_29:32
-0005.11:       OR      V_29:32 ,         0:01 ,      V_30:01
-0005.12:      STR      V_30:01 ,              ,      R_pf:01
+0005.11:       OR      V_29:32 ,         0:00 ,      V_30:00
+0005.12:      STR      V_30:00 ,              ,        pf
 0006.00:      SHL       0x1:32 ,      0x3f:32 ,      V_31:32
 0006.01:      AND      V_04:01 ,      V_31:32 ,      V_32:32
 0006.02:      SHR      V_32:32 ,      0x3f:32 ,      V_33:32
```

**Comment by radare** [comment]
cc @sushant94

**Comment by sushant94** [comment]
This is because of pf (parity flag) now being defined as a flag rather than a register and as flag_p. So either we need to change that in the above commit or I need to add a new flag type and change it to flag_p rather than pf.

**Comment by radare** [comment]
Flag_p looks less confusing than pf to me. This happened when added the new regprofile with the flags stuff defined.

Thanks

**Comment by sushant94** [comment]
I see other flags like `cf`, `zf` ... defined. Is `pf` supposed to be there or is it intentional that `pf` was removed?

**Comment by sushant94** [comment]
Ok, this can be fixed in the tests directly by using `flag_p` rather than `pf` in the esil string.
Also, it looks like pf is defined on 64 bit arch and not on 32.

**Comment by sushant94** [comment]
Fixed in fd7078f3e4268d653e30a2134ccb712881a6190c
